packages/unchained-client/src/utxo/dogecoin/parser/index.ts

```diff
@@ -0,0 +1,18 @@
+import { ASSET_REFERENCE, toAssetId } from '@shapeshiftoss/caip'
+
+import { Tx } from '../../../generated/dogecoin'
+import { BaseTransactionParser, BaseTransactionParserArgs } from '../../parser'
+
+export type TransactionParserArgs = BaseTransactionParserArgs
+
+export class TransactionParser extends BaseTransactionParser<Tx> {
+  constructor(args: TransactionParserArgs) {
+    super(args)
+
+    this.assetId = toAssetId({
```

Comment:
dogeAssetId

Reply:
addressed in https://github.com/shapeshift/lib/pull/914/commits/080c21f5f8f5aa8c96b55e31126c9b828da30d52